lib/databasehelper.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:path/path.dart';
+
+class DatabaseHelper {
```

Comment:
helper sınıfı için lib içerisine helper klasörü oluşturup oraya koyabilirsin

Reply:
Yapıldı